python-client/giskard/integrations/wandb/wandb_utils.py

```diff
@@ -1,9 +1,11 @@
 import contextlib
 from typing import Optional
 
+from wandb.wandb_run import Run
```

Comment:
I would move this:
```
    try:
        import wandb  # noqa
    except ImportError as e:
        raise ImportError("The 'wandb' python package is not installed. " "To get it, run 'pip install wandb'.") from e
```
outside `wandb_run`, and include to it `from wandb.wandb_run import Run`.

Reply:
done